Dockerfile

```diff
@@ -64,6 +64,10 @@ RUN \
 ENV FLUENCE_ENV_AQUA_IPFS_EXTERNAL_API_MULTIADDR=/dns4/ipfs.fluence.dev/tcp/5001
 ENV FLUENCE_ENV_AQUA_IPFS_LOCAL_API_MULTIADDR=/dns4/ipfs.fluence.dev/tcp/5001
 
+# connector builtin default env variable
+# instruct to deploy and start decider
+ENV FLUENCE_ENV_CONNECTOR_WITH_DECIDER=true
```

Comment:
maybe call it something like FLUENCE_ENV_JOIN_ALL_DEALS?

Reply:
The negation of the statement 'join all deals' isn't obvious. It could be 'join no deals' or 'join SOME deals' as well. I think the wording should be clear.

Reply:
the prefix should be at least `FLUENCE_ENV_CONNECTOR_`, it could be `ACCEPT_NEW_DEALS`